.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Welcome to the Julia project!
+
+We use the GitHub issue tracker for bug reports and feature requests.
+If you have a question or are unsure if the behavior you're experiencing is a bug,
+please post to [our Discourse site](https://discourse.julialang.org).
```

Comment:
>  it requires the account to be approved first, and create a new post, and wait until someone responds, is putting up way more barriers than I'd be comfortable agreeing with

Discourse can be searched without creating an account as far as I can tell, so for commonly asked questions it's not absolutely necessary to create an account (though of course we should encourage that). Perhaps we can change the `please post` wording to something like `please search or post`.